### ISSUE TITLE
Should we set expansion to zero for axis not used for alignment?

### ISSUE THREAD

**Issue by Yunuuuu** [Yunuuuu/ggalign#11]
Currently, the dendrogram is far away from the heamtap, since the ggplot2 automatically add expansion in y-axis.

![6c2085a44893d7673076bf73522c050](https://github.com/user-attachments/assets/2df817ac-37f6-478f-8420-724c09460fe0)

**Comment by ShixiangWang** [comment]
I agree, it's better (if possible) to have an argument to control this.

**Comment by Yunuuuu** [comment]
I have added a convenient function to do this, just see the `no_expansion()`, it can control the expansions in any site (top, left, bottom and right).
